Production_scripts/Process_extracts/extract_acute_incl_gls.R

```diff
@@ -258,52 +251,18 @@ outfile <- final_acute_file %>%
     disch,
     dischto,
     dischloc,
-    diag1,
-    diag2,
-    diag3,
-    diag4,
-    diag5,
-    diag6,
-    op1a,
-    op1b,
-    dateop1,
-    op2a,
-    op2b,
-    dateop2,
-    op3a,
-    op3b,
-    dateop3,
-    op4a,
-    op4b,
-    dateop4,
+    starts_with("diag"),
+    contains("op"),
     smr01_cis_marker,
     age,
-    cij_marker,
-    cij_admtype,
-    cij_ipdc,
-    cij_pattype_code,
-    cij_adm_spec,
-    cij_dis_spec,
-    CIJ_start_date,
-    CIJ_end_date,
```

Comment:
Check that these variables aren't capitalised. `CIJ_start_date`

Reply:
Good spot - only cij variables that were capitalised